scripts/west_commands/bindesc.py

```diff
@@ -0,0 +1,284 @@
+# Copyright (c) 2023 Yonatan Schachter
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from textwrap import dedent
+import struct
+
+from west.commands import WestCommand
+from west import log
+
+from elftools.elf.elffile import ELFFile
+from intelhex import IntelHex
```

Comment:
Please follow the MISSING_REQUIREMENTS pattern in runners/jlink.py to gracefully handle missing non-standard-library, non-west imports.